internal/testing/stablememmap/stablememmap.go

```diff
@@ -42,6 +42,10 @@ func (s *store) Set(ctx context.Context, c, k string, v any) error {
 	return s.mm.Set(ctx, c, k, v)
 }
 
+func (s *store) Remove(ctx context.Context, c, k string) error {
```

Comment:
@nathannaveen need to enable the backend tests to test the delete functionality.

Reply:
I forgot to push that up, thank you!